test/metrics/metrics_test.go

```diff
@@ -138,7 +138,7 @@ func TestMetricsWhenUsersManuallyApprovedAndThenDeactivated(t *testing.T) {
 	hostAwait.WaitForMetricDelta(t, wait.UserSignupsMetric, 2)                                                            // all signups (even if deactivated)
 	hostAwait.WaitForMetricDelta(t, wait.UsersPerActivationsAndDomainMetric, 2, "activations", "1", "domain", "internal") // all deactivated (but this metric is never decremented)
 	hostAwait.WaitForMetricDelta(t, wait.UsersPerActivationsAndDomainMetric, 0, "activations", "1", "domain", "external") // never incremented
-	hostAwait.WaitForMetricDelta(t, wait.UserSignupsApprovedMetric, 2)                                                    // all deactivated (but counters are never decremented)
+	hostAwait.WaitForMetricDelta(t, wait.UserSignupsApprovedMetric, 0)                                                    // all deactivated (but counters are never decremented)
```

Comment:
I'm not sure the counter will be decremented

Reply:
It did pass when I ran it locally.

Reply:
That's weird. AFAIK the "approved" metric is not supposed to decrement. @xcoulon can you confirm?

Reply:
@mmulholla  from the failure here: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/codeready-toolchain_toolchain-e2e/801/pull-ci-codeready-toolchain-toolchain-e2e-master-e2e/1708871838563569664/build-log.txt
it is failing on the parallel test suite, so it has not yet ran the metrics test suite.

I think you still have to update those conditions: https://github.com/codeready-toolchain/toolchain-e2e/blob/dfa57d215d362c64b37eb5ee14ae007e85f8ae00/test/e2e/parallel/registration_service_test.go#L375

Reply:
@mfrancisc you were, or course, correct, I removed the change to decrement the count.

Reply:
Also updated condition in registration_service_test.